xfrm_policy.go

```diff
@@ -52,6 +52,9 @@ type XfrmPolicyTmpl struct {
 type XfrmPolicy struct {
 	Dst      *net.IPNet
 	Src      *net.IPNet
+	Proto    Proto
+	DPort    uint16
```

Comment:
Makes sense, I will follow the same convention for the parameters's type. 
Agree on the names, no need to be too concise.